sample-root-scripts/read_PMT.C

```diff
@@ -49,14 +49,13 @@ void read_PMT(char *filename=NULL) {
   // As you can see, there are lots of ways to get the number of hits.
   cout << "Number of tube hits " << wcsimrootevent->GetNumTubesHit() << endl;
   cout << "Number of Cherenkov tube hits " << wcsimrootevent->GetNcherenkovhits() << endl;
-  cout << "Number of tube hits " << wcsimrootevent->GetCherenkovHits()->GetEntries() << endl;
+  cout << "Number of Cherenkov tube hits " << wcsimrootevent->GetCherenkovHits()->GetEntries() << endl;
```

Comment:
Dark noise hits are stored in this container too. Probably worth adding this to the printout for clarity (same for line 51)